### PR TITLE
docs: fixed issue with missing code sample by rearranging import to p…

### LINKED ISSUE #40558: docs: Missing code sample in Basics of testing components
# 📚 Docs or angular.io bug report

### Description

Go to: https://angular.io/guide/testing-components-basics
Go to By.css() section
You will see: `The code sample is missing for testing/src/app/banner/banner-initial.component.spec.ts#import-by`

### What's the affected URL?**
https://angular.io/guide/testing-components-basics

## 📷Screenshot

image

### PR DESCRIPTION
**Pull request by iftee-hussain**
…roper docregion

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #40558

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
Fixes #40558.

### CODE REVIEW

File: aio/content/examples/testing/src/app/banner/banner-initial.component.spec.ts [hunk 1]
```diff
@@ -1,13 +1,12 @@
 // #docplaster
 // #docregion import-by
+import { By } from '@angular/platform-browser';
 // #enddocregion import-by
```
Comment (petebacondarwin) [inline aio/content/examples/testing/src/app/banner/banner-initial.component.spec.ts:4]: I suspect that the reason this broken in the first place was that a code formatter moved the import below to put it in alphabetic order. So moving it back will probably just cause it to break again in the future. Instead, how about you move the doc-region comments to around the current location of the import?

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

File: aio/content/examples/testing/src/app/banner/banner-initial.component.spec.ts (hunk 1 above)
Comment (gkalpak) [reply aio/content/examples/testing/src/app/banner/banner-initial.component.spec.ts:4]: Also, make sure to put it outside the `v1` docregion; i.e. something like:

```diff
- // #docregion import-by
- // #enddocregion import-by // #docregion v1
 ...
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
- import { By } from '@angular/platform-browser';
 
 // #enddocregion v1
+ // #docregion import-by
+ import { By } from '@angular/platform-browser';
+ // #enddocregion import-by // #docregion v1
```

Comment (iftee-hussain) [reply aio/content/examples/testing/src/app/banner/banner-initial.component.spec.ts]: @petebacondarwin @gkalpak fixed in  3aa2ef7

**Review by petebacondarwin — APPROVED** [review]
LGTM - can you please add a line in your commit message that says:

```
Fixes #40558
```

**Review by gkalpak — APPROVED** [review]
LGTM - Thx, @iftee-hussain :+1:

Reviewed-for: global-docs-approvers

**Comment by iftee-hussain** [comment]
> LGTM - can you please add a line in your commit message that says:
> 
> ```
> Fixes #40558
> ```

done

**Comment by iftee-hussain** [comment]
> LGTM - Thx, @iftee-hussain 
> 
> Reviewed-for: global-docs-approvers

My pleasure. Thanks for the suggestions :smiley:

**Pull request closed without merging**